ibis/backends/base/sql/__init__.py

```diff
@@ -252,3 +252,9 @@ def explain(
         cur.release()
 
         return '\n'.join(['Query:', util.indent(query, 2), '', *result])
+
+    @classmethod
+    def has_operation(cls, operation: type[ops.ValueOp]) -> bool:
+        translator = cls.compiler.translator_class
+        op_classes = translator._registry.keys() | translator._rewrites.keys()
```

Comment:
We should have a special case for `ScalarParameter`

Reply:
Seems like `Pi` is not implemented for any of the backends, shall we remove it?

Reply:
`SummaryFilter` is implemented for the SQL backends but there are no tests, I assume it's unused.

Reply:
Mind creating those issues?